src/components/structures/MatrixChat.js

```diff
@@ -1118,6 +1118,7 @@ module.exports = React.createClass({
             const payload = {
                 action: 'view_room',
                 event_id: eventId,
+                // If an event ID is set (truthy), mark it as highlighted
```

Comment:
I guess I'm after something like

    // if there is an event_id in the URL, tell the room view to highlight it